advanced/lighttpd.conf.debian

```diff
@@ -46,7 +46,7 @@ compress.filetype           = ( "application/javascript", "text/css", "text/html
 
 # default listening port for IPv6 falls back to the IPv4 port
 include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
-include_shell "/usr/share/lighttpd/create-mime.assign.pl"
+include_shell "/usr/share/lighttpd/create-mime.conf.pl"
```

Comment:
This will cause any distro still using `lighttpd` v 1.4.45 or lower to not start. (Debian Stretch, Jessie, Ubuntu).

This needs to be a conditional include of either `assign` or `conf`.

Reply:
OK, I was thinking of making this conditional on the version of lighttpd (>=1.4.45 = conf, <1.4.45 = assign) but as the version number is a string, >= can't be used so instead it would probably better to drive this via the distro/release.  Do you agree before I implement?

Reply:
Actually, making this conditional will require 2x lighttpd.conf.debian files, one for Buster+ and one for older Debian/Ubuntu releases.  This may be inevitable as Ubuntu is based off Debian Testing so I imagine Ubuntu users on the latest release also have this issue

Reply:
We support Ubuntu up to 18.04 which is v1.4.45. The only Ubuntu that is greater is `disco`, which is not supported. 

https://packages.ubuntu.com/search?suite=all&searchon=names&keywords=lighttpd

Reply:
In which case I'm getting ahead of myself so will drop this PR